runtime/bindings/python/tests/test_ngraph/test_preprocess.py

```diff
@@ -0,0 +1,293 @@
+    model = parameter_a
+    function = Function(model, [parameter_a], "TestFunction")
+    layout1 = ov.Layout("NCHW")
+    layout2 = ov.Layout("NHWC")
+    layout3 = [0, 1]
+
+    @custom_preprocess_function
+    def custom_postprocess(output: Output):
+        return ops.abs(output)
+
+    function = PrePostProcessor()\
+        .input(InputInfo()
+               .tensor(InputTensorInfo().set_layout(layout1))
+               .preprocess(PreProcessSteps()
+                           .convert_element_type(Type.f32)
+                           .mean([1., 2.])
+                           )
+               ) \
+        .output(OutputInfo().postprocess(PostProcessSteps()
+                                         .convert_element_type(Type.f32)
+                                         .convert_layout(layout2)
+                                         .convert_layout(layout3)
+                                         .custom(custom_postprocess))) \
+        .build(function)
+
+    input_data = np.array([[-1, -2], [-3, -4]]).astype(np.int32)
+    expected_output = np.array([[2, 4], [4, 6]]).astype(np.float32)
+
+    runtime = get_runtime()
+    computation = runtime.computation(function)
+    output = computation(input_data)
+    assert np.equal(output, expected_output).all()
+
+
+def test_ngraph_preprocess_spatial_static_shape():
+    shape = [2, 2, 2]
+    parameter_a = ops.parameter(shape, dtype=np.int32, name="A")
+    model = parameter_a
+    function = Function(model, [parameter_a], "TestFunction")
+    layout = ov.Layout("CHW")
+
+    color_format = ColorFormat(3)
+    function = PrePostProcessor()\
+        .input(InputInfo()
+               .tensor(InputTensorInfo()
+                       .set_layout(layout)
+                       .set_spatial_static_shape(2, 2)
+                       .set_color_format(color_format, []))
+               .preprocess(PreProcessSteps()
+                           .convert_element_type(Type.f32)
+                           .mean([1., 2])
+                           )
+               .network(InputNetworkInfo().set_layout(layout))
+               ) \
+        .output(OutputInfo()
+                .tensor(OutputTensorInfo()
+                        .set_layout(layout)
+                        .set_element_type(Type.f32))
+                .network(OutputNetworkInfo().set_layout(layout))) \
+        .build(function)
+
+    input_data = np.array([[[1, 2], [3, 4]], [[5, 6], [7, 8]]]).astype(np.int32)
+    expected_output = np.array([[[0, 1], [2, 3]], [[3, 4], [5, 6]]]).astype(np.float32)
+
+    runtime = get_runtime()
+    computation = runtime.computation(function)
+    output = computation(input_data)
+    assert np.equal(output, expected_output).all()
+
+
+def test_ngraph_preprocess_steps():
+    shape = [1, 1, 3, 3]
+    parameter_a = ops.parameter(shape, dtype=np.float32, name="A")
+    model = parameter_a
+    function = Function(model, [parameter_a], "TestFunction")
+    color_format1 = ColorFormat(3)
```

Comment:
Any chance to use names like RGB/BGR for color format? It is quite inconvenient and will potentially cause breakages when new values are added
Same for 'ResizeAlgorithm'

Reply:
It should be possible with:
https://github.com/openvinotoolkit/openvino/blob/5609886b7f05842327cb252917eb562f780573ae/runtime/bindings/python/src/pyopenvino/graph/preprocess/pre_post_process.cpp#L466
@pszmel Just call to `ColorFormat.RGB`.

Reply:
Updated.